spec/unit/validator_spec.rb

```diff
@@ -381,6 +381,16 @@ def podspec_path(name = 'JSONKit', version = '1.4')
         podfile.sources.should == %w(https://github.com/CocoaPods/Specs.git)
       end
 
+      it 'avoids creation of sources when source_urls is empty and no dependencies' do
```

Comment:
Hm. I thought the idea here was that even if there were sources, we didn't need to update any of the sources if there are no dependencies, since the only spec you need is the one being linted. Maybe I'm missing something? Thanks again for all of your work on this!